rero_ils/modules/acq_orders/api.py

```diff
@@ -200,3 +203,13 @@ def bulk_index(self, record_id_iterator):
         :param record_id_iterator: Iterator yielding record UUIDs.
         """
         super().bulk_index(record_id_iterator, doc_type='acor')
+
+    def delete(self, record):
+        """Delete a record from indexer."""
+        # delete order lines from the ES index first before deleting the order.
```

Comment:
```suggestion
        # First delete order lines from the ES index, then delete the order.
```
Furthermore, could/should this comment be part of the docstring? In fact, I have this question on a much broader level: what is docstring, what is comments?